app/services/booking_response.rb

```diff
@@ -0,0 +1,9 @@
+class BookingResponse
+  def initialize(success:)
+    @success = success
+  end
+
+  def success?
```

Comment:
This class is just a stub for now, will get updated properly with the book to nomis. Do you prefer to hardcode it to return true for now?